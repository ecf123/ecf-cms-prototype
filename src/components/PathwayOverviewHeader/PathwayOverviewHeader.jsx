import "./PathwayOverviewHeader.scss";
import Button from "../Button/Button";
import { ReactComponent as Pencil } from "../../assets/pen-icon.svg";
import { Link } from "react-router-dom";
const PathwayOverviewHeader = ({ link }) => {
  return (
    <div className="pathway-header">
      <h2 className="pathway-header__title">Pathway Structure</h2>
      <Link className="pathway-header__link" to={link}>
        <Button
          style={"button black round-border medium"}
          imgStyle={"medium-img white-svg"}
          textStyle={"text medium-text black"}
          buttonText={"Add Course"}
        >
          <Pencil />
        </Button>
      </Link>
    </div>
  );
};

export default PathwayOverviewHeader;
