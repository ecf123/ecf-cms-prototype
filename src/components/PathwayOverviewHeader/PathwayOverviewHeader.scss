@use "../../styles/variables" as *;

.pathway-header {
  max-width: 56rem;
  display: flex;
  justify-content: space-between;

  &__title {
    color: $color-blue-navy-dark;
    font-size: $font-size-medium;
    font-weight: $font-weight-regular;
  }
  &__link {
    text-decoration: none;
  }
}
