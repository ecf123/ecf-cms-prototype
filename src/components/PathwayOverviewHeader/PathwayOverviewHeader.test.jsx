import PathwayOverviewHeader from "./PathwayOverviewHeader";
import { customRender } from "../../utils/testUtils";

describe("PathwayOverviewHeader component", () => {
  it("matches the snapshot taken", () => {
    const { container } = customRender(<PathwayOverviewHeader link="*" />);
    expect(container).toMatchSnapshot();
  });
});
