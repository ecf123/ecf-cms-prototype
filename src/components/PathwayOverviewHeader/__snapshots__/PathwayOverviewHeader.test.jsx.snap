// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PathwayOverviewHeader component matches the snapshot taken 1`] = `
<div>
  <div
    class="pathway-header"
  >
    <h2
      class="pathway-header__title"
    >
      Pathway Structure
    </h2>
    <a
      class="pathway-header__link"
      href="/*"
    >
      <button
        class="button black round-border medium"
      >
        <div
          class="medium-img white-svg"
        >
          <svg>
            pen-icon.svg
          </svg>
        </div>
        <p
          class="text medium-text black"
        >
          Add Course
        </p>
      </button>
    </a>
  </div>
</div>
`;
